### ISSUE TITLE
Error installing datafile on OSX using pip

### ISSUE THREAD

**Issue by dkleissa** [honnibal/spaCy#78]
I installed spacy using pip install spacy.  When I try to install the data file I get an import error:

    /usr/bin/python: cannot import name orth

Thoughts?

**Comment by honnibal** [comment]
What sequence of commands did you use? The following worked for me:

    $ virtualenv env
    $ source env/bin/activate
    $ pip install spacy
    $ python -c "
    $ python -c "import spacy.en; nlp = spacy.en.English(); print(nlp(u'hello world...').string)"
    hello world...
    $ python -m spacy.en.download all

Are you installing over the top of an old version, perhaps?

**Comment by dkleissa** [comment]
Thanks for the quick response. So that seemed to work.

I followed the commands on your Sphinx page:

    $ pip install spacy
    $ python -m spacy.en.download

When I first tried this, I immediately got an error from numpy which made me think there was some sort of binary version mismatch.  So I uninstalled spacy, and tried to follow your instructions to build from source.  That's when I started getting the import name orth error.

I restarted from scratch and and used the commands above and things seemed to work.  Thanks! Looking forward to testing this out package out.
